frontend/lib/tsconfig.json

```diff
@@ -7,8 +7,8 @@
     "emitDeclarationOnly": true,
     "declaration": true,
     "outDir": "dist",
-    "rootDir": "src",
+    "rootDir": "src"
```

Comment:
interesting that the last item's comma is removed here. right now, when I remove a last item's comma, it's automatically re-added when I press save. Did any rule change? Or is it just a local setup on my machine?

Reply:
This may have been me, but I figured JSON can't have a trailing comma?(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas#trailing_commas_in_json)

I think support is inconsistent, but you might not want it to apply to that.